cves/CVE-2014-0095.yml

```diff
@@ -204,4 +214,21 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    In my opinion, this vulnerability is not inherently a security bug. By
```

Comment:
Is it impossible for this bug to be exploited and cause a full blow denial of service attack? Are you stating here that a user can not interact with the system in a way to make use of this bug and force threads to hang?

Reply:
I agree that I was not very clear there, so I tried to clean it up a bit. Do you think my changes in commit cf508086f6399770bbd4a18b9156f5bcc52f6936 are enough?

Reply:
That looks better. The one thing I do see is that it looks like you miss typed "quite" as "qute". Other than that it looks good.